src/Angor/Shared/Services/NetworkMonitoringService.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Angor.Shared.Services;
+
+public class NetworkMonitoringService : IDisposable
+{
+    private readonly IServiceProvider _serviceProvider;
+    private Task _task;
+    private CancellationTokenSource _cancellationTokenSource;
+    private TimeSpan _checkInterval;
+    private bool _isRunning;
+
+    public NetworkMonitoringService(IServiceProvider serviceProvider)
+    {
+        _serviceProvider = serviceProvider;
+        _checkInterval = TimeSpan.FromMinutes(1); // Default check interval
+        _cancellationTokenSource = new CancellationTokenSource();
+    }
+
+    public void Start()
+    {
```

Comment:
If we have a timer it will constantly ping the services, right now we have some time delay on the check services that will only check if the last time was more then in interval of seconds, this is enough imo. a timer that will make http calls is unpredictable.

Reply:
I changed the code and removed the timer